pkg/plugin/model/repository.go

```diff
@@ -59,17 +61,31 @@ func (r *Repository) GetVulnerabilitiesSummary(ctx context.Context, options kube
 			}
 		}
 	}
-	return
+	return vs, nil
+}
+
+func (r *Repository) GetCustomResourceDefinitionByName(ctx context.Context, name string) (*v1.CustomResourceDefinition, error) {
+	unstructuredResp, err := r.client.Get(ctx, store.Key{
+		APIVersion: fmt.Sprintf("%s/%s", "apiextensions.k8s.io", "v1beta1"),
```

Comment:
Not sure there's any benefit to the Sprintf given that both the strings here are hard-coded?

Reply:
Yeah, doesn't make sense at all. I've replace it with string literal.